charts/datahub/subcharts/datahub-frontend/templates/deployment.yaml

```diff
@@ -69,6 +74,10 @@ spec:
             periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
             failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
           env:
+            {{- if .Values.global.datahub.monitoring.enablePrometheus }}
+            - name: ENABLE_PROMETHEUS
```

Comment:
Can we revert datahub-frontend? It doesn't support this yet.

Reply:
done